src/Templating/test/Templates.Test/CdnScriptTagTests.cs

```diff
@@ -96,6 +96,36 @@ public async Task FallbackSrcContent_Matches_CDNContent(ScriptTag scriptTag)
             Assert.Equal(RemoveLineEndings(cdnContent), RemoveLineEndings(fallbackSrcContent));
         }
 
+        class RetryHandler : DelegatingHandler
```

Comment:
Surprisingly, it's the tests in src/Identity/test/Identity.Test/CdnScriptTaghelperTests.cs that have been flaky in this PR. The `RetryHelper` should be helpful in any case.